contracts/GovNftSugar.vy

```diff
@@ -42,8 +43,17 @@ struct GovNft:
   address: address # address of GovNFT
   delegated: address # address of delegate
 
+struct Collection:
```

Comment:
This might need docs if we want to call a govnft release "a collection"... :pray: